aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -47,24 +54,31 @@ func testSweepElasticacheClusters(region string) error {
 
 			log.Printf("[INFO] Deleting ElastiCache Cluster: %s", id)
 			err := deleteElasticacheCacheCluster(conn, id, "")
+			if tfawserr.ErrMessageContains(err, elasticache.ErrCodeInvalidCacheClusterStateFault, "only member of a replication group") {
+				log.Printf("[WARN] ElastiCache Cache Cluster (%s) is member of a Replication Group. Ignoring. %s", id, err)
+				continue
+			}
```

Comment:
qq: if we're filtering these specific clusters out in the `DescribeCacheClustersInput` , is it still possible to hit this type of error? doesn't hurt to catch the error tho 👍

Reply:
Good catch, it's redundant since they're filtered out from the results

Reply:
gotchu, sounds good! tho i'm always cautious with AWS APIs 😅